.env.example

```diff
@@ -63,9 +63,17 @@ JMAP_TLS=True
 # MailChimp form URL for Wait List
 WAIT_LIST_FORM_ACTION=
 
+# PADDLE CLIENT TOKEN
+PADDLE_TOKEN=
+
+# PADDLE Item IDs
+PADDLE_ID_ITEM_1=
+PADDLE_ID_ITEM_2=
+PADDLE_ID_ITEM_3=
```

Comment:
Might want to mention that these are price ids, and sort order. 

In the future we might pull these directly from paddle and cache them for a period. But this is good for now, thanks!